docs/release-notes/1.1.0/release-1.1.0.md

```diff
@@ -0,0 +1,116 @@
+# [ML.NET](http://dot.net/ml) 1.1.0 
+## **New Features**
+- **Image type support in IDataView**  
+   [PR#3263](https://github.com/dotnet/machinelearning/pull/3263) added support
+  for in-memory image as a type in IDataView. Previously to consume image it was
+  impossible to directly use it in IDataView rather a user would have to specify
+  the file path as a string in IDataView and then the image would be loaded
+  using a transform. This resulted in the closure of the following github
+  issues:  [3162](https://github.com/dotnet/machinelearning/issues/3162),
+  [3723](https://github.com/dotnet/machinelearning/issues/3723),
+  [3369](https://github.com/dotnet/machinelearning/issues/3369),
+  [3274](https://github.com/dotnet/machinelearning/issues/3274),
+  [445](https://github.com/dotnet/machinelearning/issues/445),
+  [3460](https://github.com/dotnet/machinelearning/issues/3460),
+  [2121](https://github.com/dotnet/machinelearning/issues/2121),
+  [2495](https://github.com/dotnet/machinelearning/issues/2495),
+  [3784](https://github.com/dotnet/machinelearning/issues/3784) and was a much
+  requested feature by the users.  
+
+    [Sample 1](https://github.com/dotnet/machinelearning/blob/02a857a7646188fec2d1cba5e187a6c9d0838e23/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ConvertToGrayScaleInMemory.cs) | [Sample 2](https://github.com/dotnet/machinelearning/blob/02a857a7646188fec2d1cba5e187a6c9d0838e23/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMappingWithInMemoryCustomType.cs)
+
+- **Super-Resolution based Anomaly Detector**  
```

Comment:
Is this feature 1.1 GA or in preview? Please, specify.

Reply:
I have made it clear in remarks section.